tests/test_write_json.py

```diff
@@ -0,0 +1,298 @@
+"""test_write_json.py"""
+__author__ = "Marius Pahl"
+
+import json
+import pathlib
+import tempfile
+import unittest
+from collections import OrderedDict
+
+import pandas as pd
+
+from collect_stata.write_json import (
+    generate_statistics,
+    get_categorical_frequencies,
+    get_categorical_statistics,
+    get_nominal_statistics,
+    get_numerical_statistics,
+    get_univariate_statistics,
+    get_value_counts_and_frequencies,
+    write_json,
+)
+
+
+class TestWriteJson(unittest.TestCase):
+    """
+    Tests for all methods from write_json.py
+    """
```

Comment:
A one line docstrings should only be one line long.
There should be no whitespace between quotes and content.

Reply:
This applies to all following docstrings.
Also see https://www.python.org/dev/peps/pep-0257/#one-line-docstrings